packages/inter-protocol/test/psm/setupPsm.js

```diff
@@ -99,7 +99,7 @@ export const setupPsm = async (
     farZoeKit = await setUpZoeForTest();
   }
 
-  const knutIssuer = makeIssuerKit('KNUT');
+  const knutIssuerKit = makeIssuerKit('KNUT');
```

Comment:
consider `knut` (like `withAmountUtils`) for readability and ease of use
```suggestion
  const knut = withAmountUtils(makeIssuerKit('KNUT'));
```

Reply:
TypeScript was fighting me over extracting `knut.issuer`. I'll leave it for later.